code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -624,3 +624,14 @@
 	id = "laughter"
 	results = list("laughter" = 10) // Fuck it. I'm not touching this one.
 	required_reagents = list("sugar" = 1, "banana" = 1)
+
+/datum/chemical_reaction/plastic_polymers
+	name = "plastic polymers"
+	id = "plastic_polymers"
+	required_reagents = list("oil" = 5, "sodiumchloride" = 2, "ash" = 3)
+	required_temp = 374 //lazily consistent with soap & other crafted objects generically created with heat.
+
+/datum/chemical_reaction/plastic_polymers/on_reaction(datum/reagents/holder, created_volume)
+	var/location = get_turf(holder.my_atom)
+	for(var/i = 1, i <= created_volume, i++)
```

Comment:
Half of me is thinking Goof wants me to make between 10 pieces a reaction rather than 1 per small formula reagent dose as intended but that's just the paranoia talking. If that has any basis in truth Goof trust in me that the 1 per (5,2,3) ratio is deliberate.